test_unit.py

```diff
@@ -1209,6 +1230,21 @@ def test_HTLC(self):
         self.assertEqual(p, base64.b64decode(golden))
         self.assertEqual(s.get_address(), golden_addr)
 
+        golden_ltxn = (
+            "gqRsc2lngqNhcmeRxCB/g7Flf/H8U7ktwYFIodZd/C1LH6PWdyhK3dIAEm2Qaa" +
+            "FsxJcBIAToBwEAwM8kJgMg5pqWHm8tX3rIZgeSZVK+mCNe0zNjyoiRi7nJOKkV" +
+            "tvkgf4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGkg/ryguxRKWk6ntD" +
+            "ikaBrIDmyhBby2B/xWUyXJVpX2ohMxASIOMRAjEhAxBzIDEhAxCCQSEDEJKBIt" +
+            "ASkSEDEJKhIxAiUNEBEQo3R4boelY2xvc2XEIOaalh5vLV96yGYHkmVSvpgjXt" +
+            "MzY8qIkYu5yTipFbb5o2ZlZc0D6KJmdgGiZ2jEIH+DsWV/8fxTuS3BgUih1l38" +
+            "LUsfo9Z3KErd0gASbZBpomx2ZKNzbmTEIFNCP4JtpWHGzW5H9EJSwasC1THntU" +
+            "IiTJGurfnrDvCqpHR5cGWjcGF5")
+        ltxn = template.HTLC.get_transaction(
+            p, "f4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGk=", 1,
```

Comment:
I don't think this is a valid input, don't we need to pass in something which, once hashed, is equal to `f4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGk=`?